src/quacc/recipes/espresso/phonons.py

```diff
@@ -56,10 +85,192 @@ def phonon_job(
+    grid parallelization please consult the Quantum Espresso user manual and
+    exemples.
+
+    This approach requires the data of the pw.x calculation to be copied to each job,
+    leading to a total data size on the disk of n*m times the size of the pw.x calculation, where:
+    - n is the number of q-points
+    - m is the number of representations
+
+    In addition to the data produced by each ph.x calculation. This can
+    result in large data sizes for systems with many atoms.
+
+    To mitigate this, an optional "nblocks" argument can be provided. This
+    groups multiple representations together in a single job, reducing the
+    data size by a factor of nblocks, but also reducing the level of parallelization.
+    In the case of nblocks = 0, each job will contain all the representations for each q-point.
+
+    Consists of following jobs that can be modified:
+
+    1. pw.x relaxation
+        - name: "pw_job"
+        - job: [quacc.recipes.espresso.core.relax_job][]
+
+    2. ph.x calculation test_run
+        - name: "cheap_job"
+        - job: [quacc.recipes.espresso.phonons.phonon_job][]
+
+    3. (n * m) / nblocks ph.x calculations
+        - name: "ph_job"
+        - job: [quacc.recipes.espresso.phonons.phonon_job][]
+
+    4. ph.x calculation to gather data and diagonalize each dynamical matrix
+        - name: "cheap_job"
+        - job: [quacc.recipes.espresso.phonons.phonon_job][]
+
+    Parameters
+    ----------
+    atoms
+        Atoms object
+    nblocks
+        The number of representations to group together in a single job.
+        This will reduce the amount of data produced by a factor of nblocks.
+        If nblocks = 0, each job will contain all the representations for a
+        single q-point.
+    job_params
+        Custom parameters to pass to each Job in the Flow. This is a dictinoary where
+        the keys are the names of the jobs and the values are dictionaries of parameters.
+    job_decorators
+        Custom decorators to apply to each Job in the Flow. This is a dictionary where
+        the keys are the names of the jobs and the values are decorators.
+
+    Returns
+    -------
+    RunSchema
+        Dictionary of results from [quacc.schemas.ase.summarize_run][]
+    """
+
+    calc_defaults = {
+        "pw_job": {
+            "input_data": {
+                "control": {"forc_conv_thr": 5.0e-5},
+                "electrons": {"conv_thr": 1e-12},
+            }
+        }
+    }
+
+    job_params = recursive_dict_merge(calc_defaults, job_params)
+
+    # I got surprised by the 'basic_pw' that is send to pw_job implicitly,
+    # changing parameters I didn't set.
+    # I guess that users will keep forgetting about it and be surprised
+    # changing their parameters when writing custom flow?
+    pw_job, ph_test_job, ph_job, recover_ph_job = customize_funcs(
+        ["pw_job", "cheap_job", "ph_job", "cheap_job"],
+        [relax_job, phonon_job, phonon_job, phonon_job],
```

Comment:
Feel free to adjust the names as you see fit. Right now, the `ph_test_job` and `recover_ph_job` both have the same name and therefore will share the same decorator (and parameters) if specified by the user. In hindsight, I don't really think that's necessary. It probably just makes more sense to have four unique names: "pw_job", "ph_test_job", "ph_job", and "recover_ph_job". It's not any more verbose, and it's probably easier on the user. If you agree, feel free to make this change and update the names in the docstring.